ls1intum/Artemis
# Improve performance of tutor leaderboard

- Views are too slow, we now use Hibernate queries for tutor leaderboards
- This PR drops all the views in the Artemis database, which also solves a couple of other issues.
- There are still performance improvements possible, so we added time log statements to further analyze which queries are slow on production
- We deactivated the assert statements in the server test for now, but did extensive manual testing.
- We'll improve the test in a follow-up PR (now the tutor leaderboard is also easier to test, because we have no views any more).
- These changes will also make it significantly easier to create an exam tutor leaderboard 

### Checklist

### Steps for Testing
1. Open the course and exercise assessment dashboards in an existing course
2. Make sure the tutor leaderboard is displayed quickly and has the correct data
